sdk/appconfiguration/app-configuration/test/public/auth.spec.ts

```diff
@@ -31,9 +31,6 @@ describe("Authentication", () => {
   });
 
   it("token authentication works", async function() {
-    if (isPlaybackMode()) {
-      this.skip();
-    }
```

Comment:
Un-skipping the test as I don't see a reason to be skipped.

@richardpark-msft @sadasant

Reply:
If it doesn't have to be, that's awesome. I believe @sadasant did this because the recorder was giving him trouble but maybe that's resolved.

Reply:
It was because the test wasn't using the recorder provided date.
I had updated that for the test and unskipped it.